CreatubblesAPIClientIntegrationTests/Spec/Gallery/GalleriesResponseHandlerSpec.swift

```diff
@@ -38,7 +38,7 @@ class GalleriesResponseHandlerSpec: QuickSpec
             {
                 let request = GalleriesRequest(page: 1, perPage: 10, sort: .popular, userId: nil, query: nil)
                 let sender =  TestComponentsFactory.requestSender
-                waitUntil(timeout: 10)
+                waitUntil(timeout: 25)
```

Comment:
should we make this a new global timeout value? seems it changed in a number of places.